startservices.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Absolute path of the directory containing this script
+script_dir=$(dirname "$(readlink -f "$0")")
+
+# Directory to store logs
+logs_dir="$script_dir/logs"
+
+# Create the logs directory if it doesn't exist
+mkdir -p "$logs_dir"
+
+# Local username and password for SSH
+username="your_username"
+password="your_password"
+
+# Function to start the specified service
+start_service() {
+    local command="$1"
+    local service="$2"
+    local port=$(jq -r ".$service" "$script_dir/config.json" | jq -r '.port')
+    local docker_ip=$(jq -r '.docker' "$script_dir/config.json")
+    local db_port=$(jq -r '.db_port' "$script_dir/config.json")
+    local rd_port=$(jq -r '.rd_port' "$script_dir/config.json")
+    local ips=($(jq -r ".$service[]" "$script_dir/config.json"))
+
+    for ip in "${ips[@]}"; do
+        service_log="$logs_dir/${service}_${ip}_${port}.log"
+        echo "Starting $service service on $ip:$port..."
+        sshpass -p "$password" ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "$username@$ip" "bash $script_dir/runme.sh $command $port $docker_ip $db_port $rd_port" >> "$service_log" 2>&1 &
```

Comment:
Also if we had `sshpass`, `sshpass -p "$password" ...` is not secure. If this runs on the lab machines anyone can run `ps aux | grep sshpass` and be able to see the password passed in.
A better alternative is to use an *ssh key pair* ([check this out](https://www.pragmaticlinux.com/2021/05/configure-ssh-for-login-without-a-password/)) which is secure.

If you end up going this route make sure to document in the readme.